t/ui/01-list.t

```diff
@@ -109,6 +109,27 @@ sub schema_hook {
         $job99940->insert_module({name => $k, category => $k, script => $k});
         $job99940->update_module($k, {result => $v, details => []});
     }
+
+    my $schedule_job = $jobs->create(
+        {
+            id         => 99991,
+            group_id   => 1002,
+            priority   => 30,
+            result     => OpenQA::Jobs::Constants::NONE,
+            state      => OpenQA::Jobs::Constants::SCHEDULED,
+            t_finished => undef,
+            backend    => 'qemu',
+            t_started  => undef,
+            t_created  => time2str('%Y-%m-%d %H:%M:%S', time - 3600, 'UTC'),
+            TEST       => 'kde_variant',
+            ARCH       => 'x86_64',
+            BUILD      => '0092',
+            DISTRI     => 'opensuse',
+            FLAVOR     => 'NET',
+            MACHINE    => '64bit',
+            VERSION    => '13.1',
+            settings =>
+              [{key => 'JOB_TEMPLATE_NAME', value => 'kde_variant'}, {key => 'TEST_SUITE_NAME', value => 'kde'}]});
```

Comment:
here I also recommend to reuse common settings with the job from L55 from a hash to not duplicate so much.